backpack/__init__.py

```diff
@@ -142,13 +174,24 @@ def memory_cleanup(module):
         i += 1
 
 
-def hook_run_extensions(module, g_inp, g_out):
+def hook_run_extensions(
+    module: Module, g_inp: Tuple[Tensor], g_out: Tuple[Tensor]
+) -> None:
+    """The backward hook function.
+
+    It executes all BackPACK operations during the backward pass.
+
+    Args:
+        module: current module
+        g_inp: input gradients
+        g_out: output gradients
+    """
     for backpack_extension in CTX.get_active_exts():
         if CTX.get_debug():
             print("[DEBUG] Running extension", backpack_extension, "on", module)
         backpack_extension.apply(module, g_inp, g_out)
 
-    run_extension_hook(module)
+    post_extension_hook(module)
```

Comment:
[Ignore for now] Need more documentation which modules have a backward hook installed that will lead to execution of  `post_extension_hook`.